python/ambassador/ir/irratelimit.py

```diff
@@ -55,7 +57,11 @@ def setup(self, ir: 'IR', aconf: Config) -> bool:
         self.name = "rate_limit"    # Force this, just in case.
         self.namespace = config.get("namespace", self.namespace)
         self.domain = config.get('domain', ir.ambassador_module.default_label_domain)
-        self.protocol_version = config.get("protocol_version", "v2")
+
+        self.protocol_version = config.get("protocol_version", "v3")
```

Comment:
Likewise.

Reply:
this one did not need to be adjust due to no changes be made to the config. It is still using dict style.